File: php/utility/MakeContext.php
<?php
declare(strict_types=1);

// Univec SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class UnivecMakeContext
{
    public static function call(array $ctxmap, ?UnivecContext $basectx): UnivecContext
    {
        return new UnivecContext($ctxmap, $basectx);
    }
}

File: java/utility/MakeContext.java
package voxgig.univecsdk.utility;

import java.util.Map;

import voxgig.univecsdk.core.Context;

final class MakeContext {

  private MakeContext() {}

  static Context makeContext(Map<String, Object> ctxmap, Context basectx) {
    return new Context(ctxmap, basectx);
  }
}
